spec.html

```diff
@@ -4490,51 +4490,59 @@ <h2>Syntax</h2>
+          <emu-grammar>StringNumericLiteral ::: StrWhiteSpace</emu-grammar>
+          <emu-alg>
+            1. Return *+0*<sub>𝔽</sub>.
+          </emu-alg>
+          <emu-grammar>StringNumericLiteral ::: StrWhiteSpace? StrNumericLiteral StrWhiteSpace?</emu-grammar>
+          <emu-alg>
+            1. Return 𝔽(MV of |StrNumericLiteral|).
+          </emu-alg>
+          <emu-grammar>StrDecimalLiteral ::: `-` StrUnsignedDecimalLiteral</emu-grammar>
+          <emu-alg>
+            1. Let _a_ be MV of |StrUnsignedDecimalLiteral|.
+            1. If _a_ is 0, return *-0*<sub>𝔽</sub>.
+            1. Return RoundStringMVResult(-1 &times; _a_).
+          </emu-alg>
+          <emu-grammar>StrUnsignedDecimalLiteral ::: `Infinity`</emu-grammar>
+          <emu-alg>
+            1. Return *+&infin;*<sub>𝔽</sub>.
+          </emu-alg>
+          <emu-grammar>StrUnsignedDecimalLiteral ::: DecimalDigits `.` DecimalDigits? ExponentPart?</emu-grammar>
+          <emu-alg>
+            1. Let _a_ be MV of the first |DecimalDigits|.
+            1. If the second |DecimalDigits| is present, let _b_ be MV of the second |DecimalDigits|. Otherwise, let _b_ be 0.
+            1. If |ExponentPart| is present, let _e_ be MV of |ExponentPart|. Otherwise, let _e_ be 0.
+            1. Let _n_ be the number of code points in the second |DecimalDigits|.
+            1. Return RoundStringMVResult((_a_ + (_b_ &times; 10<sup>-_n_</sup>)) &times; 10<sup>_e_</sup>).
+          </emu-alg>
+          <emu-grammar>StrUnsignedDecimalLiteral ::: `.` DecimalDigits ExponentPart?</emu-grammar>
+          <emu-alg>
+            1. Let _a_ be MV of |DecimalDigits|.
+            1. If |ExponentPart| is present, let _e_ be MV of |ExponentPart|. Otherwise, let _e_ be 0.
+            1. Let _n_ be the number of code points in |DecimalDigits|.
+            1. Return RoundStringMVResult(_a_ &times; 10<sup>_e_-_n_</sup>).
+          </emu-alg>
+          <emu-grammar>StrUnsignedDecimalLiteral ::: DecimalDigits ExponentPart?</emu-grammar>
+          <emu-alg>
+            1. Let _a_ be MV of |DecimalDigits|.
+            1. If |ExponentPart| is present, let _e_ be MV of |ExponentPart|. Otherwise, let _e_ be 0.
+            1. Return RoundStringMVResult(_a_ &times; 10<sup>_e_</sup>).
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-roundstringmvresult" aoid="RoundStringMVResult">
+          <h1>RoundStringMVResult ( _n_ )</h1>
+          <p>The abstract operation RoundStringMVResult takes argument _n_ (a mathematical value). It converts _n_ to a Number in an implementation-defined manner. For the purposes of this abstract operation, a digit is significant if it is not zero or there is a non-zero digit to its left and there is a non-zero digit to its right. It performs the following steps when called:</p>
+          <emu-alg>
+            1. If the decimal representation of _n_ has fewer than 20 significant digits in its decimal representation, return 𝔽(_n_).
```

Comment:
```suggestion
            1. If the decimal representation of _n_ has fewer than 20 significant digits, return 𝔽(_n_).
```

Reply:
I think there's a fairly strong tradition that _n_, as a variable name, denotes an integer, so it might be better to better to rename `_n_` to `_x_`, say.